text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+- Start Date: 2018-07-24
+- RFC PR: 
+- Gatsby Issue: 
+
+# Summary
+
+As a follow-up to the [Site Showcase](https://next.gatsbyjs.org/showcase/), we want to create individual & organization pages highlighting the individual community members and what they've built. 
+
+We also want to create a free Job Board highlighting opportunities for employment building Gatsby websites at agencies and companies using Gatsby!
+
+# Motivation
+
+Some goals for the different components include:
+
+#### Individuals Pages
+* Highlight contributions made by community members (plugins built, code PRs)
+* "Official" landing pages to help community members in their personal website marketing efforts
+* Allow companies interested in building Gatsby sites to find individuals nearby them who are open for Gatsby projects
+
+#### Agency Pages
+* Highlight agencies building websites on Gatsby
+* "Official" landing pages to help agencies in their marketing efforts
+* Allow companies interested in building Gatsby sites to find agencies nearby them
+* Help agencies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+#### Company Pages
+* Display list of companies using Gatsby to demonstrate it's "enterprise-ready" and fulfills whatever use-case they're looking for (e-commerce, authentication, etc). 
+* Help companies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+# Detailed design
+
+## Pages
+
+#### All people / agencies / companies
+
+One template with multiple URLs, each of which apply different filters:
+  * /people/ (all people)
+  * /agencies/ (all agencies)
+  * /companies/ (all companies)
+
+Filter buttons should be at the top and should be easy to toggle between them (select-one). Filters should include:
```

Comment:
Now that I think about it, I'm having a hard time imagining what people would want to filter by. For example, would filterable criteria would a potential client even care about? Some examples of things I'd imagine they might care about a little: industry the agency typically serves, types of sites the agency specializes in, size of company/agency, size of projects or types of projects they take on, specialties, etc? 

Wondering if you already have ideas of what people would like to filter by. I think sorting will be just as powerful as filtering and easier to define (sorting by most popular, has built the most or contributed the most to Gatsby, etc).

Reply:
I'll second this. Maybe it's worth a UX project to figure out what filters people actually use?

Reply:
Project would be a bit heavyweight, but an hour or two of light research might be useful. Eg: https://www.meteor.com/partners. Drupal has a pretty crappy UI that's primarily search-based. https://www.drupal.org/organizations. Etc.